india_compliance/income_tax_india/overrides/test_asset_depreciation_schedule.py

```
@@ -0,0 +1,317 @@
+        asset.submit()
+
+        schedule = get_asset_depr_schedule_doc(asset.name, "Active", self.fb_it.name)
+        self.assertTrue(schedule.flags.get("wdv_it_act_applied"))
+
+    def test_cancel_depreciation_entries_skips_non_it_fb(self):
+        asset = self._create_asset("2024-04-01", finance_book=self.fb_regular.name)
+        asset.submit()
+
+        try:
+            cancel_depreciation_entries(asset, "2025-01-01")
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)